src/Symfony/Component/PropertyInfo/Extractor/PhpDocExtractor.php

```diff
@@ -83,6 +83,10 @@ public function getShortDescription($class, $property, array $context = [])
         }
 
         foreach ($docBlock->getTagsByName('var') as $var) {
+            if (is_a($var, 'phpDocumentor\Reflection\DocBlock\Tags\InvalidTag')) {
```

Comment:
```suggestion
            if (is_a($var, InvalidTag::class)) {
```

(add the appropriate `use` too)

Reply:
The `InvalidTag` is introduced in v5 of phpdocumentor/reflection-docblock. A use statement would be a BC break with v3 and v4 since those are autoloaded?

Reply:
no. Use statements don't imply that a class exist. And `::class` does not either (it is not an actual constant access on the class, but a compile-time thing)

Reply:
Ha I did't know that.